src/Tribe/Views/V2/View.php

```diff
@@ -793,8 +805,9 @@ public function prev_url( $canonical = false, array $passthru_vars = [] ) {
 
 		$prev_page  = $this->repository->prev()->order_by( '__none' );
 
-		$paged      = $this->url->get_current_page() - 1;
-		$query_args = $paged > 1
+		$paged           = $this->url->get_current_page() - 1;
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: c760b481f966b1acaa9f70824a2ac91c</sub>